docs/tutorials/multimodal/efficient_finetuning_basic.md

```diff
@@ -0,0 +1,142 @@
+# Parameter-Efficient Finetuning in AutoMM -- Basic Usage
+:label:`sec_automm_efficient_finetuning_basic`
+
+As pointed out by [a recent paper from Stanford Institute for Human-Centered Artificial Intelligence](https://arxiv.org/pdf/2108.07258.pdf), 
+AI is undergoing a paradigm shift with the rise of "foundation models", i.e., giant models that are trained on a diverse collection of datasets generally in a self-supervised way. 
+These foundation models, which are the key of AutoMM, can be easily adapted to down-stream applications. However, as the size of these foundation models grows, finetuning these models becomes increasingly difficult. 
+Following is a figure from the [Microsoft research blog](https://www.microsoft.com/en-us/research/blog/using-deepspeed-and-megatron-to-train-megatron-turing-nlg-530b-the-worlds-largest-and-most-powerful-generative-language-model/) that demonstrates the trend:
+
+![Model Size Scaling](https://www.microsoft.com/en-us/research/uploads/prod/2021/10/model-size-graph.jpg)
+:width:`500px`
+
+The goal of AutoMM is to democratize the publicly available foundation models, whether they are big or not, to every developers. 
+To finetune the giant models, we adopt the recently popularized **parameter-efficient finetuning** technique. 
+The idea is to either finetune a small subset of the weights in the foundation model (e.g., [BitFit](https://aclanthology.org/2022.acl-short.1.pdf)), 
+or adding a tiny tunable structure on top of the fixed backbone (e.g., [Prompt Tuning](https://aclanthology.org/2021.emnlp-main.243.pdf),
+[LoRA](https://arxiv.org/pdf/2106.09685.pdf), [Adapter](https://arxiv.org/abs/1902.00751), [MAM Adapter](https://arxiv.org/pdf/2110.04366.pdf), [IA^3](https://arxiv.org/abs/2205.05638)). 
+These techniques can effectively reduce the peak memory usage and model training time, while maintaining the performance.
+
+In this tutorial, we introduce how to apply parameter-efficient finetuning in MultiModalPredictor.
+We will reuse the same multilingual dataset as in :ref:`sec_automm_textprediction_multilingual` and train 
+models with the `"ia3_bias"` algorithm. `"ia3_bias"` is a parameter-efficient finetuning algorithm that combines IA^3 and BitFit.
+
+## Prepare Dataset
+
+The [Cross-Lingual Amazon Product Review Sentiment](https://webis.de/data/webis-cls-10.html) dataset contains Amazon product reviews in four languages. 
```

Comment:
More recent cross-lingual datasets (like XNLI) appear more frequently in recent papers, but are also harder to fit. We could also try this dataset for distillation tutorial if it takes less time to train.

Reply:
Yes, we can try this dataset.